returnn/torch/engine.py

```diff
@@ -535,14 +544,16 @@ def save_model(self):
         print("Save model under %s" % (filename,), file=log.v4)
         torch.save({"model": self._model.state_dict(), "epoch": self.epoch, "step": self._train_step}, filename)
 
-    def _load_optimizer(self, epoch):
+    def _load_optimizer(self, epoch, filename=None):
         """
         Loads a torch.optim.Optimizer from disk and uses it as the optimizer.
         This function is a wrapper to Updater.load_optimizer().
 
         :param int epoch: Epoch from which to load the optimizer state.
```

Comment:
filename param missing in docs, but do we even need both parameters? But then I think it is fine to keep it maybe, in case you really want to call it without an explicit filename. We can maybe clean later.